groupcache/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 
 mod errors;
 mod groupcache;
+pub mod groupcache_builder;
```

Comment:
I wouldn't expose it under a different mod: It should be exported under ::groupcache - there's no need to have multiple modules externally for a rather simple crate

Reply:
I'm pretty sure it would get exposed under the same ::groupcache name, but exposing it more explicitly with pub use